synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -302,6 +303,62 @@ export class SynopsysToolsParameter {
     return command;
   }
 
+  private setBlackDuckFixPrInputs(): BlackDuckFixPrData | undefined {
+    if (
+      inputs.BLACKDUCK_FIXPR_MAXCOUNT &&
+      isNaN(Number(inputs.BLACKDUCK_FIXPR_MAXCOUNT))
+    ) {
+      throw new Error(
+        "Invalid value for ".concat(constants.BLACKDUCK_FIXPR_MAXCOUNT_KEY)
+      );
+    }
+    const createSinglePr = parseToBoolean(
+      inputs.BLACKDUCK_FIXPR_CREATE_SINGLE_PR
+    );
+    if (createSinglePr && inputs.BLACKDUCK_FIXPR_MAXCOUNT) {
+      throw new Error(
+        constants.BLACKDUCK_FIXPR_MAXCOUNT_KEY.concat(
+          " is not applicable with "
+        ).concat(constants.BLACKDUCK_FIXPR_CREATE_SINGLE_PR_KEY)
+      );
+    }
+    const blackDuckFixPrData: BlackDuckFixPrData = {};
+    blackDuckFixPrData.enabled = true;
+    blackDuckFixPrData.createSinglePR = createSinglePr;
+    if (inputs.BLACKDUCK_FIXPR_MAXCOUNT && !createSinglePr) {
+      blackDuckFixPrData.maxCount = Number(inputs.BLACKDUCK_FIXPR_MAXCOUNT);
+    }
+    if (
+      inputs.BLACKDUCK_FIXPR_UPGRADE_GUIDANCE &&
+      inputs.BLACKDUCK_FIXPR_UPGRADE_GUIDANCE.length > 0
+    ) {
+      blackDuckFixPrData.useUpgradeGuidance =
+        inputs.BLACKDUCK_FIXPR_UPGRADE_GUIDANCE;
+    }
+
+    const fixPRFilterSeverities: string[] = [];
+    if (
+      inputs.BLACKDUCK_FIXPR_FILTER_SEVERITIES &&
+      inputs.BLACKDUCK_FIXPR_FILTER_SEVERITIES != null &&
+      inputs.BLACKDUCK_FIXPR_FILTER_SEVERITIES.length > 0
+    ) {
+      for (const fixPrSeverity of inputs.BLACKDUCK_FIXPR_FILTER_SEVERITIES) {
+        if (fixPrSeverity != null && fixPrSeverity !== "") {
+          fixPRFilterSeverities.push(fixPrSeverity.trim());
+        }
+      }
+      blackDuckFixPrData.filter = {
+        ...(inputs.BLACKDUCK_FIXPR_FILTER_BY
```

Comment:
we need to remove BLACKDUCK_FIXPR_FILTER_BY, not applicable now

Reply:
done